### PR TITLE
Add new status in transactions details

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

**Review by marek-siemieniuk-morawski — APPROVED** [review]

File: src/state/wallets/selectors.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ export const transactions = createSelector(wallets, electrumXSelectors.blockHeig
         let toInternalAddress;
 
         const toAddress = transaction.outputs[0].addresses[0];
-        const isToInternalAddress = wallet.weOwnAddress(toAddress);
+        const isToInternalAddress = walletsList.some(w => w.weOwnAddress(toAddress));
```
Comment (PanRadaCloudBest) [inline src/state/wallets/selectors.ts:100]: is not touch other code ?

Comment (KedziaPawel) [reply src/state/wallets/selectors.ts:100]: what do you mean? this is PR to the previous PR about the transaction list

Comment (PanRadaCloudBest) [reply src/state/wallets/selectors.ts:100]: Just question is = isToInternalAddress is not used elsewhere ? if not is ok no more questions

Comment (KedziaPawel) [reply src/state/wallets/selectors.ts:100]: no, it is not

**Pull request merged**